plugins/alignments/src/BamAdapter/BamAdapter.ts

```diff
@@ -49,8 +48,8 @@ export default class BamAdapter extends BaseFeatureDataAdapter {
           indexType !== 'CSI'
             ? openLocation(location, this.pluginManager)
             : undefined,
-        chunkSizeLimit,
-        fetchSizeLimit: 100_000_000,
+        chunkSizeLimit: Infinity,
```

Comment:
It was complaining for relatively small amounts of data from a ultra long bam file so set to infinity....the stats estimation will catch it more properly than this